homeworks/yaroslav_sakhno/hw01/test.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import argparse
+from pyats.aetest import Testcase, test, main
+from calculation import add, divide
+
+
+class TryTest(Testcase):
+
+    parameters = {
+        'num1': 3,
+        'num2': 0
+    }
+
+    @test
+    def test_add(self,steps, num1, num2):
+        with steps.start('Add'):
```

Comment:
Unnecessary step, you dont need to check is the function adds correctly

Reply:
@AlexRozl sorry, but this step is necessary for me.